src/Models/Article.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Astrogoat\Blog\Models;
+
+use Helix\Fabrick\Icon;
+use Helix\Lego\Models\Contracts\Metafieldable;
+use Helix\Lego\Models\Contracts\Sectionable;
+use Helix\Lego\Models\Model as LegoModel;
+use Helix\Lego\Models\Traits\HasMetafields;
+use Helix\Lego\Models\Traits\HasSections;
+use Spatie\Sluggable\HasSlug;
+use Spatie\Sluggable\SlugOptions;
+
+class Article extends LegoModel implements Sectionable, Metafieldable
```

Comment:
Please take a look at the [documentation](https://laravel-strata.com/docs/apps/models/#extending) to implement the `Publishable`, `Indexable`, and `Searchable` contracts. 

Please let me know if you any questions. Also, these contracts have been implemented on other models if the documentation is lacking, especially the `Page` model.